flink-test-utils-parent/flink-test-utils-junit/src/test/java/org/apache/flink/core/testutils/CommonTestUtilsTest.java

```diff
@@ -45,7 +47,29 @@ public void testObjectFromNewClassLoaderObject() throws Exception {
 
 	@Test
 	public void testObjectFromNewClassLoaderClassLoaders() throws Exception {
-		final CommonTestUtils.ObjectAndClassLoader objectAndClassLoader = CommonTestUtils.createObjectFromNewClassLoader();
+		final CommonTestUtils.ObjectAndClassLoader<Serializable> objectAndClassLoader = CommonTestUtils.createSerializableObjectFromNewClassLoader();
```

Comment:
maybe factor out one common helper test method where it does not matter whether it is `Serializable` or `Exception`?